.rpmignore

```diff
@@ -3,28 +3,25 @@
 # but should not be included in the current release
 #
 
-centos7/daos-client-tests-openmpi*.rpm
-centos7/daos-firmware*.rpm
-centos7/daos-mofed*.rpm
-centos7/daos-serialize*.rpm
-centos7/daos-server-tests-openmpi*.rpm
-centos7/daos-tests-internal*.rpm
+centos7/daos-*.rpm
 centos7/ucx*.rpm
```

Comment:
Should we remove all of the centos7/* lines here given that CentOS 7 is unsupported after 2.2?

Reply:
no, there are still some users that keep centos 7 despite it being unsupported

Reply:
This change is specific to the `master` branch.  Each `release/*` branch has it's own copy of this file.  Since we don't produce CentOS 7 RPMs on the `master` branch at all, there is nothing to ignore.  These ignore entries are therefore superfluous, dead code.